build-aux/genversion.go

```diff
@@ -39,9 +39,16 @@ func Main() error {
 	}
 	gitDescVer.Patch++
 
-	_, err = fmt.Printf("v%s-%d\n", gitDescVer, time.Now().Unix())
-	if err != nil {
-		return err
+	// If an additional arg has been used, we include it in the tag
+	if len(os.Args) >= 2 {
+		t := time.Now()
+		date := fmt.Sprintf("%d.%d.%d", t.Year(), t.Month(), t.Day())
```

Comment:
This would be better written as `date := t.Format("2006.01.02")`

Reply:
This is good to know for the future, but I'm going to remove the "date" since it's sort of irrelevant when the git commit hash is there (and actually means something)... and why add extra bloat to the tags.